modules/hello-world/examples/DocParser.java

```diff
@@ -0,0 +1,338 @@
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.lang.management.ManagementFactory;
+import java.nio.file.Files;
+import java.util.Collection;
+import java.util.Iterator;
+import java.util.LinkedList;
+import java.util.Queue;
+import java.util.Stack;
+
+public class DocParser {
+
+
+	int fileNo = 0;
```

Comment:
Please auto-reformat with soft tabs. If you can, try to get into the habit of reformatting prior to each commit. (I forget to do it sometimes too, btw).